ivy/functional/backends/tensorflow/layers.py

```diff
@@ -173,14 +173,14 @@ def depthwise_conv2d(
 @with_unsupported_devices({"2.9.1 and below": ("cpu",)}, backend_version)
 # noinspection PyDefaultArgument
 def conv3d(
-    x,
-    filters,
-    strides,
-    padding,
+    x: Union[tf.Tensor, tf.Variable],
```

Comment:
I think the conv3d() method can only accept Tensors as x and filters. Please check this out: https://www.tensorflow.org/api_docs/python/tf/nn/conv3d

Reply:
I believe tensorflow allows tf.variable to be used wherever tensors are. let me know if change is still required